paddle/fluid/operators/mkldnn/transpose_mkldnn_op.cc

```diff
@@ -124,8 +124,7 @@ class TransposeMKLDNNGradOpKernel : public paddle::framework::OpKernel<T> {
         paddle::framework::vectorize2int(out_grad->dims());
 
     const std::string key = platform::TransposeMKLDNNHandler::GetHash(
-        nchw_tz, axis, ctx.op().Output(framework::GradVarName("X")) +
-                           std::to_string(out_grad->format()));
+        nchw_tz, axis, ctx.op().Output(framework::GradVarName("X")));
```

Comment:
Why remove the `out_grad->format()` in `TransposeMKLDNNGradOpKernel`?

Reply:
@luotao1  Hi! Because in INT8 inference, we don't use `TransposeMKLDNNGradOp`(which is for backpropogation and training). @wojtuss suggests that we should not influence others' design. About why we add input->format(), please check following comment.

Reply:
Got it!